kura/org.eclipse.kura.linux.net/META-INF/MANIFEST.MF

```diff
@@ -16,6 +16,7 @@ Import-Package: javax.microedition.io,
  org.apache.commons.net.util;version="3.1.0",
  org.eclipse.kura;version="[1.1,2.0)",
  org.eclipse.kura.comm;version="[1.0,2.0)",
+ org.eclipse.kura.command;version="1.2.0",
```

Comment:
If it is importing, it should use a version range.

Reply:
Not needed. I will remove it.